content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx

```diff
@@ -147,6 +147,30 @@ const xprisma = prisma.$extends({
 })
 ```
 
+### Mutate the result of a query
```

Comment:
Are there other possible examples, besides mutating, where it would make sense to execute the `query` inside the function?